### PR TITLE
Styling Search page

### PR DESCRIPTION
**Pull request by Ayushi-0903**
Made few changes on the search page. Kindly review and suggest more changes.

Screen Shot 2019-12-07 at 1 12 47 PM

### CODE REVIEW

File: location/models.py [hunk 1]
```diff
@@ -44,16 +44,19 @@ def representative_image_or_placeholder(self):
         image = self.representative_image
         return image if image is not None else "/static/img/no_img.png"
 
-    @property
-    def rent_price_for_display(self):
+    def number_of_bed_for_display(self):
```
Comment (ab7289) [inline location/models.py]: Is there a reason why you're removing the `rent_price_for_display` method and replacing it with `number_of_bed_for_display`? I imagine that method is still useful to other parts of the code base.

File: search/templates/search/search.html [hunk 2]
```diff
@@ -193,6 +170,26 @@ <h5 class="modal-title" id="mapModalLabel">Search Result:</h5>
...
+            {% for key, percentage in search_data.stats %}
+            <li>
+                <span>{{ key }} </span><div class="meter"><span style="width: {{ percentage }}%"></span></div>
+            </li>
+            {% endfor %}
+        </ul>
+        <a href="{% url 'data_311'%}?zip_code={{zipcode}}">See more</a>
+        </div>
+    </p>
+    {% endif %}
```
Comment (ab7289) [inline search/templates/search/search.html]: It appears the the price range is no longer being displayed on the search page cards.

from `search_styling` branch:
Screen Shot 2019-12-08 at 7 16 52 PM

from develop-branch:
Screen Shot 2019-12-08 at 7 16 57 PM

File: location/models.py [hunk 3]
```diff
@@ -44,16 +44,19 @@ def representative_image_or_placeholder(self):
...
+            return self.apartment_set.first().number_of_bed_for_display
         else:
-            minimum = f"{self.apartment_set.order_by('rent_price')[0].rent_price:.0f}"
-            maximum = f"{self.apartment_set.order_by('-rent_price')[0].rent_price:.0f}"
+            minimum = (
+                f"{self.apartment_set.order_by('number_of_bed')[0].number_of_bed:.0f}"
+            )
+            maximum = (
+                f"{self.apartment_set.order_by('-number_of_bed')[0].number_of_bed:.0f}"
+            )
```
Comment (ab7289) [inline location/models.py]: It appears that we lost some test coverage as well.

Screen Shot 2019-12-08 at 7 19 58 PM

**Comment by Ayushi-0903** [comment]
To avoid unwanted changes I am creating new branch.

**Pull request closed without merging**